ArcGISRuntimeSDKQt_QMLSamples/Layers/GroupLayers/GroupLayers.qml

```diff
@@ -0,0 +1,189 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 2.5
+import Esri.ArcGISRuntime 100.5
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    SceneView {
+        id: sceneView
+        anchors.fill: parent
+
+        Scene {
+            id: scene
+            BasemapImagery {}
+
+            Surface {
+                // add an arcgis tiled elevation source
+                ArcGISTiledElevationSource {
+                    url: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"
+                }
+            }
+
+            //  Layers nested as children are appended to the group layer
+            GroupLayer {
+                id: gl
+                name: "Group: Dev A"
+
+                ArcGISSceneLayer {
+                    url: "https://scenesampleserverdev.arcgis.com/arcgis/rest/services/Hosted/DevA_Trees/SceneServer/layers/0"
+                }
+
+                ArcGISSceneLayer {
+                    url: "https://scenesampleserverdev.arcgis.com/arcgis/rest/services/Hosted/DevA_Pathways/SceneServer/layers/0"
+                }
+
+                ArcGISSceneLayer {
+                    url: "https://scenesampleserverdev.arcgis.com/arcgis/rest/services/Hosted/DevA_BuildingShell_Textured/SceneServer/layers/0"
+                }
+            }
+
+            // Add layers outside group layer
+            ArcGISSceneLayer {
+                url: "https://scenesampleserverdev.arcgis.com/arcgis/rest/services/Hosted/PlannedDemo_BuildingShell/SceneServer/layers/0"
+            }
+
+            FeatureLayer {
+                ServiceFeatureTable {
+                    url: "https://services.arcgis.com/P3ePLMYs2RVChkJx/arcgis/rest/services/DevelopmentProjectArea/FeatureServer/0"
+                }
+            }
+        }
+
+        Component.onCompleted: {
+            // create initial viewpoint extent
+            var env = ArcGISRuntimeEnvironment.createObject("Envelope", {
+                                                                json: {
+                                                                    "spatialReference": {
+                                                                        "wkid":4326
+                                                                    },
+                                                                    "xmax":-122.67960721754773,
+                                                                    "xmin":-122.68647066116789,
+                                                                    "ymax":45.53584958588318,
+                                                                    "ymin":45.531539857343745
+                                                                }
+                                                            });
```

Comment:
Same question from C++. Do you need to use json here specifically? It's more efficient to create the object by assigning the properties directly.